shared/src/main/scala/com/github/takayahilton/sqlformatter/languages/AbstractFormatter.scala

```diff
@@ -25,18 +25,40 @@ abstract class AbstractFormatter {
       FormatConfig(indent = indent, params = Params.IndexedParams(params.map(ev(_))))
     )
 
+  def formatUnsafe[A](query: String, indent: String, params: Seq[A]): String = {
+    val strParams = params.map(_.toString)
+    format(
+      query,
+      FormatConfig(indent = indent, params = Params.IndexedParams(strParams))
+    )
+  }
+
   def format[A: SqlParamable](query: String, params: Seq[A]): String =
     format(query, DEFAULT_INDENT, params)
 
+  def formatUnsafe[A](query: String, params: Seq[A]): String =
+    formatUnsafe(query, DEFAULT_INDENT, params)
+
   def format[A](query: String, indent: String, params: Map[String, A])(implicit ev: SqlParamable[A]): String =
     format(
       query,
-      FormatConfig(indent = indent, params = Params.NamedParams(params.mapValues(ev(_)).toMap))
+      FormatConfig(indent = indent, params = Params.NamedParams(params.mapValues(ev(_))))
     )
 
+  def formatUnsafe[A](query: String, indent: String, params: Map[String, A]): String = {
```

Comment:
I don't think `formatUnsafe` needs to take a type parameter.
`params: Map[String, Any]` should be enough?

Reply:
Yes indeed, I'll change that.